src/styles/index.scss

```diff
@@ -1,3 +1,125 @@
-body {
+$main-accent: #060B35;
+$secondary: #616070;
+$blue-accent: #00ACDC;
+$white: #FFF;
+$border-color: #F3F3F3;
+
+$star-size: 16px;
+
+.page__body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background: $white;
+
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  box-sizing: border-box;
```

Comment:
it's better to set for each element on the page (use `*` selector)
But note we talk only about the `box-sizing` property

Reply:
Not fixed